Entity.java

```diff
@@ -0,0 +1,74 @@
+public class Entity {
+    private double HP ;
+    private double mana;
+    private double AD;
+    private double AP;
+    private double MS;
```

Comment:
well yes since we will put those numbers into so many calculations i suggest we make the doubles and at display we round them so as to not lose values